.github/workflows/gitian.yml

```diff
@@ -0,0 +1,46 @@
+name: ci/gh-actions/gitian
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build-gitian:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system:
+          - name: "Linux"
+            option: "l"
+          - name: "Windows"
+            option: "w"
+          - name: "Android"
+            option: "a"
+          - name: "FreeBSD"
+            option: "f"
+          - name: "macOS"
+            option: "m"
+    name: ${{ matrix.operating-system.name }}
+    steps:
+    - name: prepare gitian build host 
+      run: |
+        sudo usermod -aG docker $USER
+    - name: build
+      run: |
+        curl -O https://raw.githubusercontent.com/monero-project/monero/${{ github.ref_name }}/contrib/gitian/gitian-build.py
+        chmod +x gitian-build.py
+        sudo ./gitian-build.py --setup --docker github-actions ${{ github.ref_name }}
```

Comment:
Why does this need a sudo if you've added $USER to the docker group?

Reply:
Not sure, I got a permission error without it.

Reply:
Updated and removed sudo.